README.md

```diff
@@ -1,30 +1,28 @@
-#  RHTAP Sample Backstage Templates
+# RHTAP Sample Backstage Templates
 
-
-## Software Supply Chain Security 
+## Software Supply Chain Security
 
 This repository provides a suite of language/framework examples that implement end-to-end software supply chain security (SSCS).
 
-The examples contain an advanced continuous integration pipeline covering building, CVE scanning, security scanning, signatures,  attestations, SLSA provenance and SBOM along with Gitops-based continuous deployment 
-
-Software Template authors and IDP builders can take these samples to create their own SSCS based starters. The components in the samples are designed to be maintainable to ensure that day 2 maintainance of default build pipelines and deployment yaml ensures teams have 
+The examples contain an advanced continuous integration pipeline covering building, CVE scanning, security scanning, signatures,  attestations, SLSA provenance and SBOM along with Gitops-based continuous deployment
 
+Software Template authors and IDP builders can take these samples to create their own SSCS based starters. The components in the samples are designed to be maintainable to ensure that day 2 maintainance of default build pipelines and deployment yaml ensures teams have
```

Comment:
```suggestion
Software Template authors and IDP builders can take these samples to create their own SSCS based starters. The components in the samples are designed to be maintainable to ensure 2 day maintenance of default build pipelines and deployment yaml ensures teams have.
```

> deployment yaml ensures teams have

@lcarva Does this state the deployment yaml ensures teams have something? Or should it be ensures teams have the deployment yaml they need?

Reply:
This whole sentence is a mess. I'm just gonna drop it.